wordle-web/src/App.vue

```diff
@@ -0,0 +1,11 @@
+<script setup lang="ts">
+import { RouterView } from 'vue-router'
+</script>
+
+<template>
+  <header class="text-center mb-4">
+    <h1>Wordle</h1>
```

Comment:
```suggestion
    <h1> Wordle </h1>
```
Just good practice to add this spacing, it'll help prevent weird linter formatting.